server/images.go

```diff
@@ -444,34 +444,36 @@ func paramsToReader(params map[string]string) (io.ReadSeeker, error) {
 
 	valueOpts := reflect.ValueOf(&opts).Elem()
 	// iterate params and set values based on json struct tags
-	for key, val := range params {
+	for key, vals := range params {
 		if opt, ok := jsonOpts[key]; ok {
 			field := valueOpts.FieldByName(opt.Name)
 			if field.IsValid() && field.CanSet() {
 				switch field.Kind() {
 				case reflect.Float32:
-					floatVal, err := strconv.ParseFloat(val, 32)
+					floatVal, err := strconv.ParseFloat(vals[0], 32)
 					if err != nil {
-						return nil, fmt.Errorf("invalid float value %s", val)
+						return nil, fmt.Errorf("invalid float value %s", vals)
 					}
 
 					field.SetFloat(floatVal)
 				case reflect.Int:
-					intVal, err := strconv.ParseInt(val, 10, 0)
+					intVal, err := strconv.ParseInt(vals[0], 10, 0)
 					if err != nil {
-						return nil, fmt.Errorf("invalid int value %s", val)
+						return nil, fmt.Errorf("invalid int value %s", vals)
 					}
 
 					field.SetInt(intVal)
 				case reflect.Bool:
-					boolVal, err := strconv.ParseBool(val)
+					boolVal, err := strconv.ParseBool(vals[0])
 					if err != nil {
-						return nil, fmt.Errorf("invalid bool value %s", val)
+						return nil, fmt.Errorf("invalid bool value %s", vals)
 					}
 
 					field.SetBool(boolVal)
 				case reflect.String:
-					field.SetString(val)
+					field.SetString(vals[0])
+				case reflect.Slice:
+					field.Set(reflect.ValueOf(vals))
```

Comment:
This will panic if the field is not a slice of strings. It's ok for now since there's only slices of strings but need to be revisited later